src/additional-checks.js

```diff
@@ -8,24 +8,6 @@ import {Job} from "./job.js";
  * @return {Boolean}
  */
 const validateCheck = function (checkConfig) {
-    if (typeof checkConfig !== 'object' || checkConfig === null) {
```

Comment:
We should really write some typescript :D

Reply:
Changing to typescript here is obviously out of scope for this patch. As for the definition, I would expect something more precise than `{Object}` as input definition for `checkConfig`, but I don't know much about JS docblock syntax :thinking: